src/event_loop.rs

```diff
@@ -263,6 +263,7 @@ impl<H: Handler> EventLoop<H> {
         self.io_process(handler, events);
         self.notify(handler, messages);
         self.timer_process(handler);
+        handler.tick(self);
```

Comment:
Why are you wondering this? Not following. Everything looks good to me, let me know if you want it merged :+1: